packages/website/src/components/editor/LoadedEditor.tsx

```diff
@@ -267,7 +267,7 @@ export const LoadedEditor: React.FC<LoadedEditorProps> = ({
     return debounce(() => editor.layout(), 1);
   }, [editor]);
 
-  const container = editor.getContainerDomNode?.() ?? editor.getDomNode();
+  const container = editor.getContainerDomNode();
```

Comment:
this is not correct, as this part of code is neccesary for some older versions of playground (older version of ts)

Reply:
🤔 I'm confused, where are those types defined? The `editor.api.d.ts` I see has `getContainerDomNode(): HTMLElement;` - and I thought we're settled on just one editor version?